### ISSUE TITLE
[8193.14 preview] Feedback on Sea Ships tileset tss13.set - list out of bounds/div by zero

### ISSUE THREAD

**Issue by shadguy** [Beamdog/nwn-issues#71]
Feedback from toolset evaluation:

A few tile groups under Ships cause Div by Zero/List out of Bounds errors.  Audit vs .set file.

**To Reproduce**

 - Create a new Sea Ships area in a module
 - The error can be reproduced on any of the 4 water types (Castle/City/Rrual/Tropical)
 - Expand the tile palette for "Ship/"Terrain type"\None
 - Select Boat 1 or Boat 6 or Lifeboat 2
 - You may need to repeat your section a couple times
 - Toolset typically says "Division By zero error" followed bya  bunch of "List out of Bounds" errors

**Specifics**

 - Game build 8193.14 - May 29th release
 - Windows 10)
 - Steam
 - Content overrides? (None)

**Screenshot samples**

Nnot all tile variant errors are screenshotted

![nwnee_preview_module003_Rural_None_Boat_1_ListOutofBounds](https://user-images.githubusercontent.com/38797333/83342292-58bf5000-a2b3-11ea-93ff-2d2d7dbb1a94.JPG)
![nwnee_preview_module003_Rural_None_Boat_6_ListOutofBounds](https://user-images.githubusercontent.com/38797333/83342293-5957e680-a2b3-11ea-95d5-0ab741acbef5.JPG)
![nwnee_preview_module003_Rural_None_Lifeboat_2_ListOutofBounds](https://user-images.githubusercontent.com/38797333/83342294-5957e680-a2b3-11ea-9112-5c97b86e8b40.JPG)
![nwnee_preview_module003_Tropical_None_Boat_1_ListOutofBounds](https://user-images.githubusercontent.com/38797333/83342295-5957e680-a2b3-11ea-9d78-81cb7e468e76.JPG)
![nwnee_preview_module003_Tropical_None_Lifeboat_2_ListOutofBounds](https://user-images.githubusercontent.com/38797333/83342296-59f07d00-a2b3-11ea-9842-635e27caa0db.JPG)
If warranted, add videos/screenshots to showcase the issue.

**Comment by Bill-Harper** [comment]
Still haven't found the cause, but I found a hak on Ossian SVN from 9/29/19 that doesn't seem to throw it. Worst case I think we have it covered, though I'd really like to know why as well (will investigate further).

**Comment by Bill-Harper** [comment]
I did file by file compare and only 2 lines were different; changing them didn't fix the crash. So I hunted up what files do work, and while I don't know the problem I have successfully placed every ship in all 4 water subsets.

**Comment by shadguy** [comment]
Bill, thanks for following up.  Any modelling stuff beyond some basic sanity checks (stuff showing up in AR_ERROR.,log, simple reskins, some emitter stuff) is mostly out of my wheelhouse.  I did take a look at a couple things in the .set file and they seem fine.  I did note an oddity in looking at the .itp in nwnexplorer; it turns out TLK entry 10627 ("Right Arrow") has a superfluous carriage return at the end.   I have spotted nothing else with reporting.
